core/scc/qscc/query_test.go

```diff
@@ -351,8 +354,35 @@ func addBlockForTesting(t *testing.T, chainid string) *common.Block {
 var mockAclProvider *mocks.MockACLProvider
 
 func TestMain(m *testing.M) {
+	// Read the core.yaml file for default config.
+	ledgertestutil.SetupCoreYAMLConfig()
+
+	//setup extension test environment
+	_, _, destroy := xtestutil.SetupExtTestEnv()
+	viper.Set("peer.fileSystemPath", "/tmp/fabric/core/ledger/pvtdatastorage")
+
 	mockAclProvider = &mocks.MockACLProvider{}
 	mockAclProvider.Reset()
 
-	os.Exit(m.Run())
+	code := m.Run()
+	destroy()
+	os.Exit(code)
+}
+
+func couchDBSetup() (addr string, cleanup func()) {
```

Comment:
I don't think this is ever called?

Reply:
Sorry, I didn't realize it was renamed from createTestEnv and this function is called.
Can the logic be added to the generic SetupExtTestEnv?

Reply:
I remove it